enterprise/backend/src/metabase_enterprise/metabot_v3/util.clj

```diff
@@ -21,3 +24,46 @@
   (walk/walk #(cond-> % (coll? %) (recursive-update-keys f))
              #(cond-> % (map? %) (update-keys f))
              form))
+
+;;; AI SDK support
+
+(def TYPE-PREFIX
+  "AI SDK type to prefix"
+  {:text        "0:"
+   :data        "2:"
+   :error       "3:"
+   :finish      "d:"
+   :tool-call   "9:"
+   :tool-result "a:"})
+
+(def PREFIX-TYPE "AI SDK prefix to type" (set/map-invert TYPE-PREFIX))
+
+(defn aisdk-lines->message
```

Comment:
@sloansparger really need your eyes on that function. The format I did here is just something I pulled out of thin air based on what I saw in `ai-service` code (and made a mistake with `tool-calls` vs `tool_calls` I guess, will fix that). But does this looks like something that FE will handle with no additional code? 

Does the code that renders non-streaming case still exists? Is this suitable?